content/docs/serverless/serverless-driver.md

```diff
@@ -122,9 +121,14 @@ export default async () => {
   const pool = new Pool({connectionString: process.env.DATABASE_URL});
   const db = drizzle(pool);
   const [onePost] = await db.select().from(posts).where(eq(posts.id, postId));
+  
+  pool.end();
```

Comment:
Not sure what platform this is supposed to be, but if it was Cloudflare or Vercel Edge Functions then we'd need to `ctx.waitUntil(pool.end())`, otherwise the function might get suspended before the pool is ended.

If it's just node, and the script is going to be staying alive between requests, it's OK as it is.

Reply:
changed to ctx.waitUntil(pool.end())